development/kaniko-build/README.md

```diff
@@ -36,9 +36,20 @@ cache:
   cache-run-layers: true
 ```
 
+## Build multi-architecture images
+
+>You are reading about experimental feature. Keep in mind that there may be some changes in the future.
+
+With introduction of experimental BuildKit support, the tool now supports repeatable flag `--platform`.
+You can define multiple architectures on which image should be built.
```

Comment:
... on which an/ the image ... depending on whether it is a specific image or not.

Reply:
Edited